cfgen/dhall/types/Ports.dhall

```diff
@@ -0,0 +1,26 @@
+{-
+  Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  For any questions about this software or licensing,
+  please email opensource@seagate.com or cortx-questions@seagate.com.
+
+-}
+
+-- network ports
+{ hax : Optional Natural
+, m0_ios : Optional Natural
+, m0_client_s3 : Optional Natural
```

Comment:
@SwapnilGaonkar7 . @mssawant ,
Considering RGW, should we have some change here instead of s3?

Reply:
@suvratjoshi, that will be done in a separate task and is out of scope of this task.